next.config.js

```diff
@@ -8,7 +8,6 @@ const withTM = require('next-transpile-modules')([
 ])
 
 module.exports = withTM({
-  target: 'serverless',
```

Comment:
next recommended removing this when i ran `yarn test` https://nextjs.org/docs/messages/deprecated-target-config

Reply:
:+1: